packages/dlp/package.json

```diff
@@ -1,12 +1,16 @@
 {
   "repository": "GoogleCloudPlatform/google-cloud-node",
-  "name": "dlp",
-  "version": "0.0.0",
+  "name": "@google-cloud/dlp",
+  "version": "0.7.1",
```

Comment:
The version is the gapic version. This is something we should sort out: should we attach GAPIC-only API client library version to the gapic version or should we feel free to manually overwrite?